roles/get-sitename/tasks/main.yml

```diff
@@ -128,7 +128,7 @@
 #
 - name: Manipulate sites.json
   set_fact:
-    sites_formatted: "{{ sites_formatted|default([]) + [ {'url': item.0, 'id': sites_json.sites[item.0].conf.acsf_site_id, 'site': item.0|regex_replace('.(dev-|test-)cardinalsites.acsitefactory.com', '') } ] }}"
+    sites_formatted: "{{ sites_formatted|default([]) + [ {'url': item.0, 'id': sites_json.sites[item.0].conf.acsf_site_id, 'site': item.0|regex_replace('.(dev-|test-|)cardinalsites.acsitefactory.com', '') } ] }}"
```

Comment:
For the site key we don't want the full url and was missing the or option for production to string the remaining part of the url.

Reply:
ONE FREAKING PIPE???

Now I'm _extra_ irritated that nobody laughed at my regexp joke today.

Reply:
I laughed and then I wept while the gentle chirping of crickets sounded through my headphones.